build/bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=devworkspace-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=fast
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.7.1+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-unknown
```

Comment:
I love `go get` for providing unknown. We need to take a look at what we can do with it before merging.

Reply:
bingo do go build, but it does not matter:
```bash
$ go get github.com/operator-framework/operator-sdk/cmd/operator-sdk@v1.7.2
$ operator-sdk version                                             10s 09:58:16
  operator-sdk version: "unknown", commit: "unknown", kubernetes version: "unknown", go version: 
  "go1.16", GOOS: "linux", GOARCH: "amd64"
```